blog/_posts/2017-02-22-repl-0.6-highlights.md

```diff
@@ -0,0 +1,108 @@
+
+Finding the relevant information can be quite difficult, especially the file name and line numbers since they are not aligned in any way.
+In 0.6, this is what is currently showen for the same error:
+
+![Stacktrace in 0.6](https://github.com/JuliaLang/julialang.github.com/blob/master/blog/_posts/repl0.6/0.6stacktrace.png)
+
+We stopped printing the contents of the `Array` on a `BoundsError` but more relevant for this section, the way stacktraces are shown is significantly different.
+They are no longer written in bold red, which helps distinguish it from the error message and the parts that are most likely to be interesting (the function name and the line info) are now bold, making them stick out from the surrounding text.
+Every stackframe is also numbered which makes them easier to refer to.
+Furthermore, is is now possible to customize the colors that the functions and line infos are printed with.
+This is done by setting the environment variables `JULIA_STACKFRAME_FUNCTION_COLOR` and `JULIA_STACKFRAME_LINEINFO_COLOR` respectively.
+Executing:
+
+```julia
+julia> ENV["JULIA_STACKFRAME_LINEINFO_COLOR"] = :cyan;
+
+julia> ENV["JULIA_STACKFRAME_FUNCTION_COLOR"] = :yellow;
+```
+
+and rerunning the example, now displays:
+
+![Stacktrace in 0.5 with configured colors](https://github.com/JuliaLang/julialang.github.com/blob/master/blog/_posts/repl0.6/0.6stacktrace_conf.png)
+
+In addition to the visual changes of the stacktrace, there is also a new interactive feature, possible now that we can refer to stackframes by number.
+After a stacktrace has been printed, you can simply enter the number of one of the stack frames in the REPL, press `CTRL + Q` and the file will be opened in
+the editor at the file and line number relevant to that stackframe.
+To set the editor, use the environment variable `EDITOR`, for example `ENV["EDITOR"] = "subl"` for the Sublime Text editor.
+This can be used when you quickly want to fix a typo or just to jump around with the editor in the call stack leading up to the error.
+
+Note that it is currently not possible to open a piece of code defined in the REPL in an editor.
+
+## More colors
+
+In v0.5, it was possible to use 8 colors for printing (see `help?> Base.text_colors`) by using the function `print_with_color`, which  unconditionally wrote the colored text in bold.
+In 0.6, light versions of these colors (prefixed by `light_`) are also supported, increasing the number of named colors to 16.
+In addition, an integer between 0 and 255 can now be used as a color (see [this chart](https://commons.wikimedia.org/wiki/File:Xterm_256color_chart.svg) for a reference).
+These new colors are possible to use wherever an option previously existed to change the color of some element in the REPL.
+Another change is that by default, the function `print_with_color` no longer prints in bold.
+Instead a new keyword argument `bold::Bool` determines if the text is printed in bold or not.
```

Comment:
Technically, these improvements have nothing to do with the REPL.  You can use colored/bold output in non-interactive scripts, and they also work in IJulia or any other environment that recognizes ANSI color codes.